common_test.go

```diff
@@ -0,0 +1,33 @@
+package bitmarkdClient
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+func setup() error {
+	if err := os.Mkdir("test-certs", 0755); err != nil {
+		return err
+	}
+
+	cmd := exec.Command("openssl",
+		"req", "-new", "-nodes", "-x509", "-out", "test-certs/server.pem", "-keyout", "test-certs/server.key",
+		"-days", "3650", "-subj", "/C=TW/ST=TPE/L=Earth/O=Bitmark Inc/OU=IT/CN=www.bitmark.com/emailAddress=test@bitmark.com")
+	return cmd.Run()
+}
+
+func tearDown() {
+	os.RemoveAll("test-certs")
+}
+
+func TestMain(m *testing.M) {
```

Comment:
what's the purpose of this test?

Reply:
This function is made to test preparation.